newIDE/app/src/UI/Slideshow/Slideshow.js

```diff
@@ -108,15 +104,18 @@ const Slideshow = ({
   // Ensure the component is re-rendered when the window is resized.
   useOnResize(useForceUpdate());
   const windowInnerWidth = window.innerWidth;
+  const windowInnerHeight = window.innerHeight;
 
   const classesForArrowButtons = useStylesForArrowButtons();
 
-  const windowWidth = useResponsiveWindowWidth();
-  const isMobile = windowWidth === 'small';
+  const { isMobile } = useResponsiveWindowSize();
+  const isLandscape = windowInnerWidth > windowInnerHeight;
```

Comment:
Can you use the one coming from userResponsiveWindowSize?

Reply:
yes forgot to rename this one